code/index.html

```diff
@@ -1,30 +1,136 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <link rel="icon" type="image/png" href="./resources/images/favicon.ico">
+  <title>Kitten Transfer</title>
+
+</head>
+<body>
+  <main class="main">
+    <picture class="bg-image">
+      <source srcset="./resources/images/ella-jardim-mKWKbHZaN-0-unsplash-1000.jpg" media="(min-width: 640px)">
+      <source srcset="./resources/images/ella-jardim-mKWKbHZaN-0-unsplash-1920.jpg" media="(min-width: 1000px)">
+      <source srcset="./resources/images/ella-jardim-mKWKbHZaN-0-unsplash-2400.jpg" media="(min-width: 1200px)">
+      <img class="hero-image" src="./resources/images/ella-jardim-mKWKbHZaN-0-unsplash-640.jpg" alt="">
+    </picture>
+    <div class="video-container">
+      <video id="bg-video" class="bg-video" autoplay loop muted>
+        <source src="./resources/videos/Kitten-1080.mp4" type="video/mp4" media="(min-width: 720px)">
+        <source src="./resources/videos/Kitten-1080.webm" type="video/webm" media="(min-width: 720px)">
+        <source src="./resources/videos/Kitten-720.mp4" type="video/mp4" media="(min-width: 480px)">
+        <source src="./resources/videos/Kitten-720.webm" type="video/webm" media="(min-width: 480px)">
+        <source src="./resources/videos/Kitten-480.mp4" type="video/mp4">
+        <source src="./resources/videos/Kitten-480.webm" type="video/webm">
+
+          <p>This browser doesn't support video...</p>
+      </video>
+    </div>
+    <div class="content-container">
+      <div class="top-full-width">
+        <!-- Top-logo -->
+        <div class="top-logo">
+          <p class="logo-text">Kitten Transfer</p>
+        </div>
+        <ul class="nav">
+          <li>About us</li>
+          <li>Products</li>
+          <li>Sign in</li>
+        </ul>
```

Comment:
This could be a `nav` and a container could be a `header` instead of a `div` in a more semantical approach